### ISSUE TITLE
cannot use static ldap secrets, only dynamic

### ISSUE THREAD

**Issue by ninjadude333** [hashicorp/vault-secrets-operator#187]
**Describe the bug**
trying to fetch a static ldap secret, a rotated password for an existing ldap user, this is not working, dynamic ldap secrets do work.

**To Reproduce**
```
vault write ldap/static-role/hashicorp \
    dn='uid=hashicorp,ou=People,dc=ninjadude,dc=com' \
    username='hashicorp' \
    rotation_period="6h"


---
apiVersion: secrets.hashicorp.com/v1alpha1
kind: VaultDynamicSecret
metadata:
  namespace: default
  name: example
spec:
  vaultAuthRef: example
  mount: ldap   **(Also tried ldap/static-role)**
  role: hashicorp
  destination:
    create: true
    name: dynamic1


**Expected behavior**
the ldap credential should be fetched.

**Environment**
* EKS 1.24
vault ent 
vault-secrets-operator version: latest

**Comment by benashz** [comment]
@ninjadude333 I believe we have fixed this in https://github.com/hashicorp/vault-secrets-operator/pull/172 (unreleased). We have replace `role` with `path` so that you can specify any `path` relative to the secret engine's mount.

**Comment by ninjadude333** [comment]
great thanks. is there an eta for release?

**Comment by benashz** [comment]
We don't have an exact date yet. It should be just after [v0.1.0-beta.1](https://github.com/hashicorp/vault-secrets-operator/milestone/1) is closed.

**Comment by benashz** [comment]
Fixed in #172
